app/src/main/java/com/tip/lunchbox/view/listeners/RecyclerTouchListener.java

```diff
@@ -0,0 +1,52 @@
+package com.tip.lunchbox.view.listeners;
```

Comment:
@epicadk wouldn't simply implementing `RecyclerItemClickListener.OnItemClickListener` inside the view and specify the actions inside `onItemClick` method been simpler and easier?

Reply:
We'll have to add it inside the Search View and Home View so this will reduce the repetition of code that's why I created a class.

Reply:
As far as I know, still that would have been simpler and less in terms of boilerplate

Reply:
Yes but going forward if we want to implement long clicks and swipes this would be more efficient is what I thought while implementing it.

Reply:
Long-press can again be handled easily. Yeah swipe actions may require some work but I don't think so you can handle them using `GestureDetector.SimpleOnGestureListener` also
Also, I don't think we shall need swipe actions on item. If you have any ideas then please do share

Reply:
I thought other gestures would let a user do things like quick add restaurant to favorites or something similar.

Reply:
Nice idea,but can't we simply use long press for that?
For now, I think simple itemClickListener will suffice. Later if we plan to have such complex use cases,  we will adopt your approach.